index.md

```diff
@@ -13,6 +13,6 @@ Get started by:
 
 ### Frequently Asked Questions
 
-You can [search through all entries on GitHub](https://github.com/issues?q=archived%3Afalse+user%3Asafing+sort%3Aupdated-desc+label%3Afaq).
+You can your browser search on [this page]({{ site.faq_url }}) or [search through all entries on GitHub](https://github.com/issues?q=archived%3Afalse+user%3Asafing+sort%3Aupdated-desc+label%3Afaq).
```

Comment:
wording is not right here ;) but push to `master` for this small fix after I merged